lib/components/DateRangePicker.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import * as dayjs from "dayjs";
+import { DateRangePicker as BPDateRangePicker } from "@blueprintjs/datetime";
+
+const DateRangePicker = React.forwardRef(
+  (
+    {
+      minDate = dayjs("1960-01-01", "YYYY-MM-DD").toDate(),
+      maxDate = dayjs("2050-01-01", "YYYY-MM-DD").toDate(),
+      dayPickerProps,
+      ...otherProps
+    },
+    ref
+  ) => {
+    return (
+      <>
+        <BPDateRangePicker
+          ref={ref}
+          allowSingleDayRange={true}
```

Comment:
For a date range picker, in most cases, we would expect 2 dates. So we don't need to set this to true by default.